tools/run-clangcldocker.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+COMMAND=$@
```

Comment:
nit: This should [technically](https://www.shellcheck.net/wiki/SC2124) be `$*`

Reply:
Modified.